src/deepsparse/pipeline.py

```diff
@@ -139,8 +139,9 @@ def create(cls, task: str, **kwargs) -> "Pipeline":
                     "Pipeline was not created for the given task. The "
                     "provided task should be registered using the OperatorRegistry"
                 )
-        except Exception:
-            _LOGGER.warning(f"Could not create v2 '{task}' pipeline, trying legacy")
+        except Exception as e:
+            _LOGGER.warning(f"Could not create v2 '{task}' pipeline, with error: {e}")
+            _LOGGER.warning(f"Attempting to create the legacy pipeline")
```

Comment:
could we combine these to one string so only one message is logged?

Reply:
Yeah I just figured the error might be long. But I'll put it at the end, something like:
```python
_LOGGER.warning(f"Could not create v2 '{task}' pipeline, attempting to create the legacy pipeline. V2 pipeline error: {e}")
```

Reply:
You can just do:

```
_LOGGER.warning(f"Could not create v2 '{task}' pipeline, with error: {e}. "
                                 "Attempting to create the legacy pipeline")
```

Reply:
I think Luka meant the error will be long so the sentence might look weird in the console

Reply:
Yeah just in case